dmoj/executors/autoconfig.py

```diff
@@ -29,35 +36,45 @@ def main():
         if executor is None or not hasattr(executor, 'Executor'):
             continue
 
+        if silent:
+            # if you are printing errors into stdout, you may do so in your own blood
+            # *cough* Racket *cough*
+            Executor = type('Executor', (executor.Executor, NullStdoutMixin), {})
```

Comment:
Are you on drugs? You need to tell the code below to actually use this class!!!!